src/lib/chat_manager.ts

```diff
@@ -209,36 +275,38 @@ data.watch("config.ogs", (settings) => {
 });
 */
 
-
-
 export function resolveChannelDisplayName(channel: string): string {
     if (channel === "shadowban") {
         return global_channels[channel];
     }
     if (channel.startsWith("global-")) {
-        global_channels.forEach(element => {
+        global_channels.forEach((element) => {
             if (channel === element.id) {
                 return element.name;
             }
         });
     } else if (channel.startsWith("tournament-")) {
         const id: number = parseInt(channel.substring(11));
-        tournament_channels.forEach(element => {
+        tournament_channels.forEach((element) => {
             if (id === element.id) {
                 return element.name;
             }
         });
     } else if (channel.startsWith("group-")) {
         const id: number = parseInt(channel.substring(6));
-        group_channels.forEach(element => {
+        group_channels.forEach((element) => {
             if (id === element.id) {
                 return element.name;
             }
         });
     } else if (channel.startsWith("game-")) {
-        return interpolate(_("Game {{number}}"), {"number": channel.substring(5)});
+        return interpolate(_("Game {{number}}"), {
+            number: channel.substring(5),  // eslint-disable-line
+        });
     } else if (channel.startsWith("review-")) {
-        return interpolate(_("Review {{number}}"), {"number": channel.substring(7)});
+        return interpolate(_("Review {{number}}"), {
+            number: channel.substring(7),  // eslint-disable-line
```

Comment:
Here's a change that triggers eslint's `ban-names` rule.  ESLint wasn't detecting `"number": ...`, but it does detect `number: `

Not hard to work around (just use a name like `num_reviews` instead), but a change to be aware of.

Reply:
Interesting, yeah it's not that big of a deal I don't suppose, but modifying this causes translation churn as the whole string needs to be re-translated. It looks like there are only a few cases of "number" being used in translations though, so I reckon it's fine to just bite the bullet at some point and have some bad translations for a bit until they're patched.

Reply:
>modifying this causes translation churn as the whole string needs to be re-translated.

I should have clarified that the workaround I suggested was for future translations.  For existing translations, we could avoid churn by invoking `eslint-disable-line id-denylist` and maybe a clarifying comment.